### PR TITLE
Feature #r53 add map component larysa shashuk

### PR DESCRIPTION
**Pull request by LarysaShashuk**
dev
## ClickUp

https://app.clickup.com/t/c50j4u

## Summary of change

- Map component was added to the user cabinet

## Testing approach

react testing library
jest

### CODE REVIEW

File: src/components/forCabinetPage/options/univListOption/univListOption.module.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 .universityListSection {
   width: inherit;
   height: auto;
-  background-color: $white;
+  background-color: #f2f9ff;
```
Comment (andrew-melnychuk) [inline src/components/forCabinetPage/options/univListOption/univListOption.module.scss]: $lightBlue_3

Comment (LarysaShashuk) [reply src/components/forCabinetPage/options/univListOption/univListOption.module.scss]: fixed

**Review by Rominos7 — APPROVED** [review]
Good

**Review by andrew-melnychuk — APPROVED** [review]
good

**Review by Romanarkko — APPROVED** [review]
Approve

**Pull request merged**